src/main/resources/db/changelog/changes/07-insert-users-table.yaml

```
@@ -0,0 +1,21 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-users-table
+      author: dmytro

```

```suggestion
      author: dmytro_potapov
```
it's better to provide full name as author of changeSet